pkg/parser/model/ddl.go

```diff
@@ -787,6 +788,20 @@ func (job *Job) DecodeArgs(args ...interface{}) error {
 	return nil
 }
 
+// DecodeDropIndexFinishedArgs decodes the drop index job's args when it's finished.
+func (job *Job) DecodeDropIndexFinishedArgs() (
```

Comment:
We are trying to define a struct for each type of DDL jobs (see job_args.go in https://github.com/pingcap/tidb/pull/55854), this will be removed, FYI

Reply:
yes, please merge master, use job version 2

Reply:
Currently merge master results in a lot of commits and code changes that are not part of this PR. I will merge master after this PR.